### PR TITLE
P-2378 missing color props for checkbox row component

### PR DESCRIPTION
**Pull request by gunjankothari**

### CODE REVIEW

File: packages/core/src/components/Checkbox/CheckboxRow.tsx [hunk 1]
```diff
@@ -184,5 +195,23 @@ export const SEED_DATA = {
       valuePropName: "status",
       handlerPropName: "onPress",
     }),
+    color: createColorProp({
+      group: GROUPS.basic,
```
Comment (peterpme) [inline packages/core/src/components/Checkbox/CheckboxRow.tsx:199]: you can remove group, label, defaultValue from these - the defaults you set are the defaults of the function

**Review by peterpme — CHANGES_REQUESTED** [review]
everything else is good besides the small seed data changes, nice job!

File: packages/core/src/components/Checkbox/CheckboxRow.tsx [hunk 2]
```diff
@@ -184,5 +195,20 @@ export const SEED_DATA = {
...
+      label: "Unselected Color",
+      description: "Color for the button when the checkbox is unchecked",
+      defaultValue: null,
+    }),
+    indeterminateColor: createColorProp({
+      group: GROUPS.basic,
+      label: "Indeterminate Color",
+      description: "Color for the button when the checkbox is indeterminate",
+      defaultValue: null,
```
Comment (dhruska) [inline packages/core/src/components/Checkbox/CheckboxRow.tsx:211]: Peter's comment applies to `uncheckedColor` and `indeterminateColor`, too - `defaultValue` and `group` are the same as the defaults applied in `createColorProp`, so they can be removed here.

Comment (gunjankothari) [reply packages/core/src/components/Checkbox/CheckboxRow.tsx]: fixed

**Review by dhruska — APPROVED** [review]

**Pull request merged**